packages/replayer/src/output.utils.ts

```diff
@@ -77,13 +79,30 @@ const writeRawCoverageData: (options: {
   outputDir: string;
   coverageData: CoverageEntry[];
 }) => Promise<void> = async ({ outputDir, coverageData }) => {
+  // We already snapshot the assets, so don't need to save the script contents
+  const withoutScriptText = coverageData.map(
+    ({ text: _text, ...rest }) => rest // eslint-disable-line @typescript-eslint/no-unused-vars
+  );
+  const withoutMeticulousScriptCoverage = withoutScriptText.filter(
+    ({ url }) => !isMeticulousSnippetURL(url)
+  );
   await writeFile(
     join(outputDir, "raw-coverage.json"),
-    JSON.stringify(coverageData, null, 2),
+    JSON.stringify(withoutMeticulousScriptCoverage, null, 2),
     "utf-8"
   );
 };
 
+const isMeticulousSnippetURL = (url: string) => {
+  try {
+    return new URL(url).origin === METICULOUS_SNIPPET_ORIGIN;
```

Comment:
Could potentially use `getSnippetsBaseUrl` instead of redefining the snippet origin here - will probably require moving that util function to the common package though. Up to you if you want to do the refactoring as part of this.

Reply:
Ah yeah this is a good call / worth doing. I'll update.